src/main/java/com/homework/pageobjects/LoginPage.java

```diff
@@ -11,48 +12,50 @@
 @PageObject
 public class LoginPage {
 
-  private static final String URL =  "https://mail.google.com/mail/?pc=topnav-about-en#inbox";
+    @Inject
+    @Named("inbox.url")
+    private String inboxURL;
 
-  @Inject
-  private WebDriver webDriver;
+    @Inject
+    private WebDriver webDriver;
 
-  @Inject
-  private BobcatWait bobcatWait;
+    @Inject
+    private BobcatWait bobcatWait;
 
-  @FindBy(id ="identifierId")
-  private WebElement userNameField;
+    @FindBy(id = "identifierId")
+    private WebElement userNameField;
 
-  @FindBy(id = "identifierNext")
-  private WebElement identifierNext;
+    @FindBy(id = "identifierNext")
+    private WebElement identifierNext;
 
-  @FindBy(css = "input[name='password']")
-  private WebElement passwordField;
+    @FindBy(css = "input[name='password']")
+    private WebElement passwordField;
 
-  @FindBy(id="passwordNext")
-  private WebElement passwordNext;
+    @FindBy(id = "passwordNext")
+    private WebElement passwordNext;
 
-  @FindBy(css = "div[aria-live='assertive']")
-  private WebElement inncorrectLoginMessage;
+    @FindBy(css = "div[aria-live='assertive']")
+    private WebElement inncorrectLoginMessage;
```

Comment:
This works but is a sucky way to find this element. Investigate if there is a better way to find this element